packages/deprecated/src/test/index.js

```diff
@@ -19,23 +19,23 @@ describe( 'deprecated', () => {
 		deprecated( 'Eating meat' );
 
 		expect( console ).toHaveWarnedWith(
-			'Eating meat is deprecated and will be removed.'
+			'Eating meat is deprecated.'
 		);
 	} );
 
 	it( 'should show a deprecation warning with a version', () => {
 		deprecated( 'Eating meat', { version: 'the future' } );
 
 		expect( console ).toHaveWarnedWith(
-			'Eating meat is deprecated and will be removed in the future.'
+			'Eating meat is deprecated and will be removed in version the future.'
```

Comment:
This doesn't seem like something we'd want, nor what would be expected. Might be what's failing the build 😄

Reply:
Build fails temporarily because of the known issue with annotations test.

I think this unit test is a bit misleading.

If you provide a number, it makes more sense:

`Eating meat is deprecated and will be removed in version 12.0.`

Reply:
Oh, I see. Maybe we should update the test cases to avoid it reading so awkwardly?

Reply:
Yep, I hesitate to do it in the first place but I will do it now 👍

Reply:
https://github.com/WordPress/gutenberg/pull/16774/commits/0a048a30cf999ce8859224163aa2c3a08c85f176 - addressed